packages/lesswrong/lib/collections/posts/custom_fields.ts

```diff
@@ -177,7 +179,8 @@ addFieldsDict(Posts, {
   isRead: resolverOnlyField({
     type: Boolean,
     viewableBy: ['guests'],
-    resolver: async (post, args, { ReadStatuses, currentUser }: { ReadStatuses: CollectionBase<DbReadStatus>, currentUser: DbUser }) => {
+    resolver: async (post, args, context: ResolverContext) => {
+      const { ReadStatuses, currentUser } = context;
```

Comment:
not PR-review-blocking, but for my knowledge: can you literally not use deconstructions at the same time you use ResolverContext, or is just cleaner looking?

Reply:
You can use deconstructions. Which one is cleaner looking is pretty close to a toss up, and I don't feel strongly either way.

Reply:
(Note, I much prefer deconstructions in the arguments, but not blocking)